controllers/logoutController/index.js

```diff
@@ -0,0 +1,11 @@
+const passport = require('passport');
+require('../../passportConfig')(passport);
+
+function logOut(req, res){
+    req.logout();
+    //Send a response to the front end to clear local storage and redirect to login page.
```

Comment:
Is this comment referring to the line above? If so they should be switched so that the comment is above the line of code.